src/components/interactive/ButtonGroup.md

```diff
@@ -3,10 +3,10 @@ Hardcoded example without any local state
 ```js
 import ButtonGroup  from './ButtonGroup';
 
-import FeedIcon from 'fds-icons/lib/react/FeedIcon';
-import TableIcon from 'fds-icons/lib/react/TableIcon';
-import MarketMapIcon from 'fds-icons/lib/react/MarketMapIcon';
-import StarEmptyIcon from 'fds-icons/lib/react/StarEmptyIcon';
+import FeedIcon from '../../../lib/icons/react/FeedIcon';
```

Comment:
I'm not the biggest fan of relative paths like this... I'm generally only a fan of relative paths things that are located together (so this markdown file and the component are together, so it makes sense to use relative paths) but for things completely different, I find it introduces a lot of finickiness, doesn't do a great job communicating (developer importing this depending on how deep their folder structure is in their given use case will have to use trial and error each time drilling up), and could be more consistent.

There should be a way to refer to these things "absolutely" IMO, e.g.

```
import Blah from '/lib/blah/blah'
```

That way, anything someone needs to consume something inside this package, it's a uniform path, similar to what we had before when they were actual packages.

There's a way to set this up with either babel or webpack tweaks if I recall correctly.